plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.mediawiki

```diff
@@ -219,13 +232,30 @@ One additional tool is provided by the Viewpoint:
 
 Several Layers are provided for Cybersecurity PAB, PDFB, CDB and ES diagrams. A layer allows to change the presentation of a diagram in order to focus on specific aspects of the modeled system. Layers are activated in the diagram toolbar:
 
-[[Image:Images/layers.png]]
+[[Image:Images/layers(new).png]]

```

Comment:
move layesr(new) to layers, do not create (new) files, just update the old ones.